packages/hooks/useTrackEnrollment.js

```diff
@@ -0,0 +1,26 @@
+import { useMutation } from 'urql'
+import enrollInTrackMutation from '@hylo/graphql/mutations/enrollInTrackMutation'
+import leaveTrackMutation from '@hylo/graphql/mutations/leaveTrackMutation'
+
+export default function useTrackEnrollment () {
+  const [enrollResult, enrollInTrack] = useMutation(enrollInTrackMutation)
+  const [leaveResult, leaveTrack] = useMutation(leaveTrackMutation)
+
+  const handleEnrollInTrack = async (trackId) => {
+    const result = await enrollInTrack({ trackId })
+    return result?.data?.enrollInTrack?.id != null
+  }
+
+  const handleLeaveTrack = async (trackId) => {
+    const result = await leaveTrack({ trackId })
+    return result?.data?.leaveTrack?.id != null
+  }
+
+  return {
+    enrollInTrack: handleEnrollInTrack,
```

Comment:
this seems like a weird pattern to me, having a use hook return actions. why wouldn't we just import those directly where they are needed?